actions/epp_render_remote.yaml

```diff
@@ -0,0 +1,85 @@
+---
+name: epp_render_remote
+runner_type: remote-shell-script
```

Comment:
I'm not sure this is going to work. `epp_render.py` includes this line:

```python
from lib.remote_actions import PuppetBaseAction
```

When ST2 runs `remote-shell-script`, it copies the script across to the remote host, but I don't think it will be able to pick up dependencies?

Reply:
Ran the following:
```
$ st2 run puppet.lookup_remote hosts=puppet.dev.encore.tech username=serviceaccount password=xxx keys='encore::ntp::servers'
...
id: 5b7b0eeba814c012e405f85a
status: succeeded
parameters: 
  hosts: puppet.dev.encore.tech
  keys:
  - encore::ntp::servers
  password: '********'
  username: root
result: 
  puppet.dev.encore.tech:
    failed: false
    return_code: 0
    stderr: "Running command "/opt/puppetlabs/bin/puppet lookup --render-as=json encore::ntp::servers""
    stdout:
    - time1.domain.tld
    - time2.domain.tld
    succeeded: true
```

This is the same way that the other actions within this pack work. Digging through the code for the remote runner i found this: https://github.com/StackStorm/st2/blob/master/contrib/runners/remote_runner/remote_runner/remote_script_runner.py#L99-L109

That may explain why it's working?

Reply:
Interesting - didn't realize it did that. Looks like it's OK then.